sql/core.go

```diff
@@ -540,6 +541,12 @@ type TriggerDatabase interface {
 	DropTrigger(ctx *Context, name string) error
 }
 
+// TemporaryTableDatabase is a database that can query the session (which manages the temporary table state) to
+// retrieve the name of all temporary tables.
+type TemporaryTableDatabase interface {
```

Comment:
Get rid of this, no need for this interface

Just iterate over all tables and ask each one if it's temp or not

Reply:
Resolved in DMs